database/getters.go

```diff
@@ -33,3 +41,45 @@ func (g *Get) AllTableNames() ([]string, error) {
 
 	return tableNames, nil
 }
+
+func (g *Get) TableColumns(t string) ([]TableColumn, error) {
+	queryString := fmt.Sprintf("PRAGMA table_info(%s);", t)
```

Comment:
I am sorry. It appears that we dont need to use Prepare here, i went through the docs again and it seems Prepare is only used while inserting and other stuff. We can use the normal query here. We can merge this and i can make the change. Sorry for that

Reply:
No problem. I also researched a bit and it seems that this only makes sense when reusing the query for performance improvements, safety etc. Sounds good!